dotcom-rendering/src/web/components/BrazeMessaging.importable.tsx

```diff
@@ -0,0 +1,22 @@
+import { log } from '@guardian/libs';
+import { useBraze } from '../lib/useBraze';
+
+type Props = {
+	idApiUrl: string;
+};
+
+/**
+ * This component ensure we call buildBrazeMessaging at least once
+ * on every page
+ */
+export const BrazeMessaging = ({ idApiUrl }: Props) => {
+	// eslint-disable-next-line no-void -- we don’t use this promise
```

Comment:
I think because of the line below (`if (brazeMessages) ...`) we shouldn't need this eslint-disable?